x-pack/plugins/security_solution/server/lib/telemetry/tasks/timelines.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { Logger } from '@kbn/core/server';
+import { SafeEndpointEvent } from '../../../../common/endpoint/types';
+import { ITelemetryEventsSender } from '../sender';
+import { ITelemetryReceiver } from '../receiver';
+import type { TaskExecutionPeriod } from '../task';
+import type {
+  ESClusterInfo,
+  ESLicense,
+  TimelineTelemetryTemplate,
+  TimelineTelemetryEvent,
+} from '../types';
+import { TELEMETRY_CHANNEL_TIMELINE } from '../constants';
+import { resolverEntity } from '../../../endpoint/routes/resolver/entity/utils/build_resolver_entity';
+
+export function createTelemetryTimelineTaskConfig() {
+  return {
+    type: 'security:telemetry-timelines',
+    title: 'Security Solution Timeline telemetry',
+    interval: '3h',
+    timeout: '10m',
```

Comment:
Interesting -- do you have any idea how long this task typically runs in?

Reply:
~2-4 seconds for 100 alerts on a one node ES test cluster. Both on a local network. 
In deployments, it depends on the Elasticsearch cluster responsiveness.